src/main/java/duke/model/ModelManager.java

```diff
@@ -0,0 +1,91 @@
+package duke.model;
+
+import duke.commons.exceptions.DukeException;
+import duke.logic.CreateMap;
+import duke.model.events.Event;
+import duke.model.events.Task;
+import duke.model.locations.BusStop;
+import duke.model.locations.Venue;
+import duke.model.transports.BusService;
+import duke.storage.Storage;
+import javafx.collections.transformation.FilteredList;
+import javafx.collections.transformation.SortedList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ModelManager implements Model {
+    private Storage storage;
+    private TaskList tasks;
+    private CreateMap map;
+    //private List<BusStop> allBusStops;
+    //private List<TrainStation> allTrainStations;
+    //private List<Route> userRoutes;
+
+    /**
+     * Constructs a new ModelManager object.
+     */
+    public ModelManager() {
+        storage = new Storage();
+        tasks = storage.getTasks();
+        map = storage.getMap();
+        //allBusStops = storage.getBusStops();
+        //allTrainStations = storage.getTrainStations();
```

Comment:
don't retain commented out code. if you want this code fragment later, use VCS to retrieve it.